documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -0,0 +1,311 @@
+# Setting Up Security for a Distributed Mode Cluster
+
+In the Distributed Mode of running a cluster, each component runs separately on one or more processes, usually distributed across multiple machines. The deployment options in this mode include:
+
+* A manual deployment in hardware or virtual machines
+* Containerized deployments of these types:
+  * A Kubernetes native application deployed using the Pravega Operator
+  * A Docker Compose application deployment
+  * A Docker Swarm based distributed deployment
+
+Regardless of the deployment option used, setting up Transport Layer Security (SSL/TLS) for as well as client authentication and authorization (`Auth` in short) are important steps towards a secure deployment. TLS enables encryption of client-server and inter-component traffic and the clients to authenticate the server. Client `auth` enables the servers to authenticate and authorize the clients. Pravega strongly recommends always enabling security for production clusters.
+
+Setting up security - especially TLS - in a large cluster can be daunting at first. To make it easier, this document provides step-by-step instructions on how to enable security when deloying a Pravega cluster manually.
+
+Depending on the deployment option used and your environment, you might need to modify the steps and commands  to suite your specific needs and policies. Also, some of the deployment options open up additional ways of securing a cluster, as we will see in the next sub-section; We'll not discuss those ways in detail in this document, for the sake of brevity.
```

Comment:
Modify the steps and commands based on your deployment option and environment.

Reply:
`Also, some of the deployment options open up additional ways of securing a cluster, as we will see in the next sub-section; We'll not discuss those ways in detail in this document, for the sake of brevity.`
can be
Also, some of the deployment options open up additional ways of securing a cluster, as those are covered in the next sub-section and not in the scope of this document.

Reply:
Fixed.